junit4/README.md

```diff
@@ -0,0 +1,215 @@
+# Weld JUnit 4 Extension
+
+Weld JUnit 4 extension uses the original JUnit extension system via `@Rule` annotation.
+It requires JUnit 4.9+ and Java 8.
+
+## Table of contents
+
+* [JUnit 4](#junit-4)
+  * [WeldInitiator](#weldinitiator)
+    * [Flat Deployment](#flat-deployment)
+    * [Convenient Starting Points](#convenient-starting-points)
+      * [Test class injection](#test-class-injection)
+      * [Activating context for a normal scope](#activating-context-for-a-normal-scope)
+      * [Adding mock beans](#adding-mock-beans)
+
+## JUnit 4
```

Comment:
This should be probably `Maven Artifact` or something like that. And `WeldInitiator` section should be on the same level.